src/Application/Clipboard/ClipboardItem.cc

```diff
@@ -126,6 +126,10 @@ void ClipboardItem::resize( size_t width, size_t height, Core::DataType data_typ
   case Core::DataType::UINT_E:
     buffer_size = sizeof( int );
     break;
+  case Core::DataType::LONGLONG_E:
```

Comment:
At least, this commit should be called `Dženan's and Lee's changes`. But I prefer you cherry-picking all our commits with their messages (possibly squashing some, but not all commits).

Reply:
Oh I'm so sorry! I didn't mean to not give credit to Lee! I really appreciate the help. I will cherry pick in the future!